gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -84,6 +84,101 @@
 Default list of subsets to include in the sample QC meta HT from the project meta HT
 """
 
+CURATED_SUBPOPS = {
```

Comment:
I don't really like having these here, I think they would be better in the subpops script

Reply:
yeah i was back and forth on this, i'll move it to subpop_analysis.py and import here instead